pydm/widgets/axis_table_model.py

```diff
@@ -69,6 +69,8 @@ def get_data(self, column_name, axis):
             return axis.auto_range
         elif column_name == "Log Mode":
             return axis.log_mode
+        elif column_name == "Hidden":
+            return not axis.isVisible()
```

Comment:
Was the intent to surface this in designer? I'm not sure there's too much of a use case for that, but this change won't do anything on its own.

Reply:
There is now a "hidden" checkbox button that the user can click to hide/show an axis (and all of its attached curves).

Reply:
It is necessary, but I don't seen any "hidden" checkbox yet. If you need it to show up in Designer, you'd need to add it here as well:

https://github.com/slaclab/pydm/blob/8fff1fa566afdb249d4d5bdca21530540a313f1a/pydm/widgets/axis_table_model.py#L14-L21

and here:

https://github.com/slaclab/pydm/blob/8fff1fa566afdb249d4d5bdca21530540a313f1a/pydm/widgets/axis_table_model.py#L92-L115

Unless the checkbox you are talking about is on the archiver viewer GUI and not PyDM, in which case this probably isn't needed.

Reply:
Yes, sorry I should have mentioned that. It is on the Archive Viewer side. I can migrate it here if people would like it to be there, or if it makes things less confusing? I just don't want to add a button to people's plots without them knowing/asking for it.

Reply:
And by that logic, if the setter is on the archive viewer but the getter is in pydm, I should keep it consistent. I will therefore move one to the other side - so then it's up to you, if you think it should all be on the Archive Viewer to not mess with other plots, or all on PyDM side if you think it's a useful feature for everyone to have access to.

Reply:
It can be left there, but then these 2 lines shouldn't be needed for that. `axis_table_model.py` specifically defines the table for editing plot axes in PyQt Designer (screenshot below). I wouldn't think your changes in the archive viewer would need anything changed here, unless I'm missing something.

![Screenshot from 2024-08-30 16-18-26](https://github.com/user-attachments/assets/8dd946d8-200e-4f78-a957-22e564e6ef29)

Reply:
Yeah, I'd keep them both on Archive Viewer then

Reply:
Sounds good. Done